File: MODUL 4 PEMROGRAMAN 1/PRAK405-2310817120002-SITI RATNA DWINTA SARI.py
for u in range(3):
    nil, klp = map(int, input().split())
    totalsm = 0

    for i in range(1, nil + 1):
        totbar = 0
        for a in range(i, 0, -1):
            hper = a * klp
            totbar += hper
            print(f"({a} * {klp})", end="")
            if a > 1:
                print(" + ", end="")
        totalsm += totbar
        print(f" = {totbar}")

    print(f"{totalsm}\n")

File: MODUL 4 PEMROGRAMAN 1/PRAK405-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>

int main() {
    int u, i, a, nil, klp, totalsm, totbar, hper;
    for (u = 0; u < 3; u++) {
        scanf("%d %d", &nil, &klp);
         totalsm = 0;

        for (i = 1; i <= nil; i++) {
            totbar = 0;
            for (a = i; a >= 1; a--) {
                hper = a * klp;
                totbar += hper;
                printf("(%d * %d)", a, klp, hper);
                if (a > 1) {
                    printf(" + ");
                }
            }
            totalsm += totbar;
            printf(" = %d\n", totbar);
        }

        printf("%d\n\n", totalsm);
    }
    return 0;
}
